core/src/com/javaciv/client/ClientView.java

```diff
@@ -581,4 +607,48 @@ public void closeAllSelectedTiles() {
         selectedTiles.clear();
     }
 
+
+    public void Leftclicked(float x, float y){
+        System.out.println("Click at [" + x + ", " + y + "]");
+        int clickedX = (int) getMouseCoordinates().x; 
+        int clickedY = (int) getMouseCoordinates().y;
+        if (isCityClicked(clickedX, clickedY)) {
+            System.out.println("City clicked at coordinates: [" + clickedX + ", " + clickedY + "]");
+        } else {
+            System.out.println("No city clicked. Coordinates: [" + clickedX + ", " + clickedY + "]");
+        }
+    }
+
+    public boolean isCityClicked(int x, int y) {
```

Comment:
Alors ça c'est pas bien y'a 2 problème dont un qui est majeur (si je dis pas de bétise) : 
- Itérer sur toute les villes qui existe pour trouver la bonne c'est pas ouf imo mais ça en vrai chill c'est pas très grave
- Tu itére que sur les villes qui t'appartiennent quand tu fais ça, le jour ou tu clique sur une ville qui n'est pas à toi tu vas pas avoir ce que tu veux.

Il faudrait faire un champ en plus sur l'objet tuille qui comprends une ville, comme ça on fait un tile.getCity